src/service/event-service.ts

```diff
@@ -252,7 +253,7 @@ export default class EventService {
   public static async getSingleEvent(id: number): Promise<EventResponse | undefined> {
     const event = await Event.findOne({
       where: { id },
-      relations: ['createdBy', 'shifts', 'answers', 'answers.user'],
+      relations: { createdBy: true, shifts: { roles: true }, answers: { user: true } },
```

Comment:
maybe kinda out of scope of this PR? I'll allow it though.

Reply:
I did add an extra relation to be returned (`shifts.roles`), so I thought "why then not also immediately fix the ugly array syntax" so here we are.